rbe3002/PointArray bounding_points
---